### ISSUE TITLE
GridX: load of ellipsis.css fails in certain cases

### ISSUE THREAD

**Issue by aerwin** [maqetta/maqetta#2172]
I don't believe this will ultimately require any changes from the Maqetta team (e.g., Evan's gridx team will need to resolve), but opening so don't lose track of the issue.

In two cases related to use of GridX, the following shows up in the browser console:

```
GET http://localhost:8081/maqetta/app/static/lib/gridx/dojox/html/resources/ellipsis.css 404 (ProxyServlet: /maqetta/app/static/lib/gridx/dojox/html/resources/ellipsis.css)
```

These two cases are:
1. Opening the gridx wizard
2. loading a user page with a an instance of GridX if the user has previously changed the location of the gridx library

Basically, It looks like one of the GridX css files has a hard-coded relative path that only works if the gridx lib is a peer to the dojo lib. I've pointed this out to Evan, and agreed to look into it.

**Comment by peller** [comment]
@evanhw

**Comment by aerwin** [comment]
> M7

**Comment by evanhw** [comment]
Hi aerwin,

It's already fixed with https://github.com/evanhw/gridx/commit/886ee6d79b81a0abd6fbed363f92d97e4716029a

Thanks for finding this!

**Comment by aerwin** [comment]
> M6

**Comment by aerwin** [comment]
Thanks, @evanhw ! I just verified the fix, so closing this issue.

**Comment by aerwin** [comment]
Reopening... reappeared after upgrading to GridX 1.0 for issue #2568 (commit e7e3825578b1307edf5c4cb64817180069e8ab94 ). Note has been sent to @evanhw for investigation.

**Comment by aerwin** [comment]
@evanhw fixed this for me in GridX code. I pointed at this fix by updating submodules in Maqetta commit 51ec4c6526540e08774f6102a24351658489a7af . So, closing again.
